.gitignore

```diff
@@ -14,4 +14,4 @@ hs_err_pid*
 .idea/
 *.iml
 .gradle/
-
+build/
```

Comment:
ok, checked and I have definitely not added the build folder ... all I have done (and this is what the line in question above does) is add the entry to the .gitignore file so that it will not be added accidentally in the future.